gapic/templates/testing/constraints-3.7.txt.j2

```diff
@@ -11,7 +11,7 @@ protobuf==3.19.5
 {% for package_tuple, package_info in pypi_packages.items() %}
 {# Quick check to make sure the package is different from this setup.py #}
 {% if api.naming.warehouse_package_name != package_info.package_name %}
-{% if api.requires_package(package_tuple) %}
+{% if api.requires_package(package_tuple) or (api.has_iam_mixin and package_tuple == ('google', 'iam', 'v1')) %}
```

Comment:
Do we not need this logic in the other `constraints-3.{8,9,10,11}.txt.j2` files? Even if it's not being exercised at the moment (and arguably it should be), we're going to move the 3.7 functionality to one of those as 3.7 reaches EOL, correct?

(Also see my other comment about not hard-coding the mixin explicitly here.)

Reply:
Good catch, this issue was fixed in https://github.com/googleapis/gapic-generator-python/pull/1631/commits/b3307ded78bf33763818d29be62242b0c020ebb0